CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Changed
 
 - Replaced `Png` images with `Svg` images for Clumit theme.
+- Added the `role` attribute to the `radio` and `checkbox` elements implemented
+  with `div` tags, which will enhance web accessibility.
```

Comment:
I think it would better to include this in the `Fixed` section.

Reply:
I have moved that part to the Fixed section.